### DISCUSSION TITLE
What languages are supported?

### DISCUSSION THREAD

**Discussion by ghost** [coqui-ai/TTS#693] [General Q&A]
The readme says that the TTS supports 20+ languages, but which?

**Comment by erogol** [comment]
Which one do you need ?

**Reply by ghost** [reply]
I'm curios to know if Bulgarian is supported.
Anyway, it would be nice if a list was available somewhere in the readme or the docs.

**Reply by erogol** [reply]
There is no limit in languages if you want to train models with raw characters. 

The only limit is when you want to train a model with phonemes. Then we only support Japanese, Chinese, and the languages listed here https://github.com/rhasspy/gruut#supported-languages

**Reply by shi-yan** [reply]
I'm curious if I want to train a model for chinese, do I need to implement a word separator first? do the model expect the words are separated in a sentence?

Any pretrained models for other languages?

**Reply by erogol** [reply]
you can see all the models by

```bash
tts --list_models
```

I've never trained Chinese model myself before but I hope someone will answer the question.
